arangograph/certificates.md

```diff
@@ -0,0 +1,155 @@
+- default port `8840`
+- high port `18840`
+
+Metrics are accessible on two different port numbers:
+- default port `8829`
+- high port `18829`
+
+The distinction between these port numbers is in the certificate used for the
+TLS connection.
+
+### Well-known X.509 certificates
+
+**Well-known X.509 certificates** created by
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"} are used on the
+default ports, i.e. `8529`.
+
+This type of certificate has a lifetime of 5 years and is rotated automatically.
+It is recommended to use well-known certificates, as this eases access of a
+deployment in your browser.
+
+{% hint 'info' %}
+The well-known certificate is a wildcard certificate and cannot contain
+Subject Alternative Names (SANs). To include a SAN field, which is needed
+for private endpoints, please use the self-signed certificate.
+{% endhint %}
+
+### Self-signed X.509 certificates
+
+**Self-signed X.509 certificates** are used on the high ports, i.e. `18529`.
+This type of certificate has a lifetime of 1 year, and it is created by the
+ArangoGraph platform. It is also rotated automatically before the expiration
+date.
+
+{% hint 'info' %}
+Unless you switch off the **Use well-known certificate** option in the
+certificate generation, both the default and high port serve the same
+self-signed certificate.
+{% endhint %}
+
+### Subject Alternative Name (SAN)
+
+The Subject Alternative Name (SAN) is an extension to the X.509 specification 
+that allows you to specify additional host names for a single SSL certificate.
+
+When using [private endpoints](deployments.html#how-to-create-a-private-endpoint-deployment),
+you can specify alternate domain names. Note that these are added **only** to
+the self-signed certificate as Subject Alternative Name (SAN).
+
+## How to create a new certificate
+
+1. Click a project name in the **Projects** section of the main navigation.
+2. Click the **Security** tab.
+3. In the **Certificates** section, click:
+   - The **New certificate** button to create a new certificate.
+   - A name or the **eye** icon in the **Actions** column to view a certificate.
+     The dialog that opens provides commands for installing and uninstalling
+     the certificate through a console.
+   - The **pencil** icon to edit a certificate.
+     You can also view a certificate and click the **Edit** button.
+   - The **tag** icon to make the certificate the new default.
+   - The **recycle bin** icon to delete a certificate.
+
+![ArangoGraph Create New Certificate](images/arangograph-new-certificate.png)
+
+Certificates that have the **Use well-known certificate** option enabled do
+not need any installation and are supported by almost all web browsers
+automatically.
+
+When creating a self-signed certificate that has the **Use well-known certificate**
+option disabled, the certificate needs to be installed on your local machine as
+well. This operation slightly varies between operating systems.
```

Comment:
```suggestion
well. This operation varies between operating systems.
```

Point out that this is described in the UI?

Reply:
It is already mentioned above in How to create a new certificate.